src/utils/url.js

```diff
@@ -177,6 +177,31 @@ export function parseQueryString(query) {
 }
 
 
+/**
+ * Adds a parameter to a query string.
+ * @param {string} url
+ * @param {string} param
+ * @param {string} value
+ * @return {string}
+ */
+export function addQueryParam(url, param, value) {
+  const queryIndex = url.indexOf('?');
+  const fragmentIndex = url.indexOf('#');
+  let fragment = '';
+  if (fragmentIndex != -1) {
+    fragment = fragmentIndex != -1 ? url.substring(fragmentIndex) : '';
```

Comment:
this line is executed only if `fragmentIndex != -1`
so could just set: `fragment = url.substring(fragmentIndex);`

Reply:
Thanks! Missed it. Fixed.